elasticsearch/index_templates/common.settings.kibana.template.json

```diff
@@ -4,5 +4,5 @@
     "index.number_of_replicas": $REPLICA_SHARDS,
     "index.number_of_shards": $PRIMARY_SHARDS
   },
-  "template": ".kibana*"
+  "index_patterns": ".kibana*"
```

Comment:
does this have any relation to the kibana term "index pattern"? or is it just overloaded at this point...?

Reply:
It is overloaded atm.
They changed the field name from `template` to `index-patterns` which is probably better name as it now accepts also list of strings (not only a single string value like `template` did).

See:
- 5.6 https://www.elastic.co/guide/en/elasticsearch/reference/5.6/indices-templates.html
- 6.x https://www.elastic.co/guide/en/elasticsearch/reference/6.8/indices-templates.html (although a single string value is still accepted)
- 7.x https://www.elastic.co/guide/en/elasticsearch/reference/7.8/indices-templates-v1.html#put-index-template-v1-api-request-body (in 7.x the docs is precise in terms of allowed data types)